onnxruntime/python/tools/transformers/onnx_exporter.py

```diff
@@ -49,8 +49,12 @@ def restore_torch_functions():
 
 
 def create_onnxruntime_input(vocab_size, batch_size, sequence_length, input_names, config, data_type=numpy.int64):
-    input_ids = numpy.random.randint(low=0, high=vocab_size - 1, size=(batch_size, sequence_length), dtype=data_type)
+    if config.model_type=="vit":
```

Comment:
May add another function like create_ort_image_input(batch_size, data_type=numpy.float32).
This function seems for text input task.